### PR TITLE
Fixing broken tags in readme

### PR DESCRIPTION
**Pull request by rivkahstandig3636**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -599,7 +599,6 @@ Default: `[]`
 </dt>
 <dd>If `True`, suppresses the warning normally shown when `rollbar.init()` is called multiple times. Default `False`.
 </dd>
-</dl>
```
Comment (brianr) [inline README.md]: Weird - there is a starting `<dl>` on line 384.

**Comment by rokob** [comment]
https://github.com/rollbar/pyrollbar/commit/e797bd429db4701d07e3abd3c3e6f76f09b99d7c#diff-04c6e90faac2675aa89e2176d2eec7d8

**Comment by rokob** [comment]
The `<dl></dl>` tags are supposed to be there, they are just mismatched.

**Comment by rivkahstandig3636** [comment]
Hm I see. I'm on a call but I'll go back to this right after.

**Pull request merged**